apps/sms/js/utils.js

```diff
@@ -445,12 +445,18 @@
         return null;
       }
       var mainPart = mime.slice(0, index);
+      var secondPart = mime.substr(index + 1);
+
       switch (mainPart) {
         case 'image':
           return 'img';
+        case 'text':
+          if (secondPart !== 'plain') {
+            return 'ref';
+          }
+          return mainPart;
```

Comment:
I think it is clearer. Actually, I did it in response to a Julien's comment in the initial feedback he have me